setup.py

```diff
@@ -113,11 +112,12 @@
         "rich>=10.14,<11",
         "tqdm>=4.62.3,<5.0",
         "typing-extensions ; python_version<'3.8'",
-        "web3[tester]>=5.29.0,<6.0",
-        "eth_abi==2.1.1",
-        "eth-utils==1.10.0",
-        "eth-rlp==0.2.1",
         "python-dateutil>=2.8.2,<3.0",
+        "web3[tester]>=6.0.0b2,<7.0",
```

Comment:
I didn't mean to push these changes yet.. May undo. I was seeing if it help wit the `<?>` but it hasn't.

I really want to upgrade web3.py to the 6.0 range but its still in beta so may wait.

- [ ] Either upgrade to released 6.0.0 or undo these changes